Packs/MalwareParser/Scripts/Parse/MalwareFindings.py

```diff
@@ -0,0 +1,161 @@
+
+    def _parse_context(self) -> None:
+        self._parse_kill_chain(self.search_value)
+
+
+class Persistence(KillChain):
+    def __init__(self, context: dict):
+        super().__init__(name='EvidenceOfPersistence',
+                         question='Is there evidence of Persistence?',
+                         context=context,
+                         search_value=ThreatIntel.KillChainPhases.PERSISTENCE)
+
+
+class DefenseEvasion(KillChain):
+    def __init__(self, context: dict):
+        super().__init__(name='EvidenceOfDefenseEvasion',
+                         question='Is there evidence of Defense Evasion?',
+                         context=context,
+                         search_value=ThreatIntel.KillChainPhases.DEFENSE_EVASION)
+
+
+class Execution(KillChain):
+    def __init__(self, context: dict):
+        super().__init__(name='EvidenceOfExecution',
+                         question='Is there evidence of Execution?',
+                         context=context,
+                         search_value=ThreatIntel.KillChainPhases.EXECUTION)
+
+
+class LateralMovement(KillChain):
+    def __init__(self, context: dict):
+        super().__init__(name='EvidenceOfLateralMovement',
+                         question='Is there evidence of Lateral Movement?',
+                         context=context,
+                         search_value=ThreatIntel.KillChainPhases.LATERAL_MOVEMENT)
+
+
+class PrivilegeEscalation(KillChain):
+    def __init__(self, context: dict):
+        super().__init__(name='EvidenceOfPrivilegeEscalation',
+                         question='Is there evidence of Privilege Escalation?',
+                         context=context,
+                         search_value=ThreatIntel.KillChainPhases.PRIVILEGE_ESCALATION)
+
+
+class CommandAndControl(KillChain):
+    def __init__(self, context: dict):
+        super().__init__(name='EvidenceOfCommandAndControl',
+                         question='Is there evidence of Command and Control?',
+                         context=context,
+                         search_value=ThreatIntel.KillChainPhases.COMMAND_AND_CONTROL)
+
+
+class FileSignature(MalwareFinding):
+    def _parse_context(self) -> None:
+        self.additional_attributes = {}
+        self.status = Status.SUSPICIOUS  # default
+
+        # MSDE
+        self._parse_single('MicrosoftATP.File.IsValidCertificate', 'true', Status.BENIGN, Source.TODO)  # todo True?
+
+        if signer := demisto.get(self.context, 'MicrosoftATP.File.Signer'):
+            self.additional_attributes['Signer'] = signer
+            self.sources.add(Source.TODO)  # todo
+
+        # WildFire
+        if publisher := demisto.get(self.context, 'File.DigitalSignature.Publisher'):
+            self.status = Status.BENIGN
+            self.sources.add(Source.TODO)  # todo
+            self.additional_attributes['Signer'] = publisher
+            # todo what if both signer and publisher are found? return list? concatenate?
```

Comment:
I guess the name is confusing 
the intention is of course who signed the file 
and that we see in File.DigitalSignature.Publisher